test/Tests.Infrastructure/CommonTestsAssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+#if DEBUG || VALIDATE || TEST_PERFORMANCE
```

Comment:
only `TESTING`

Reply:
Ok. I will remove the DEBUG from there once everything else will be in place. No?